shared/src/business/useCases/processStreamRecordsInteractor.js

```diff
@@ -329,11 +397,23 @@ exports.processStreamRecordsInteractor = async (
       useTempBucket: false,
     });
 
+  const getMessage = ({
```

Comment:
Does this method need to be marked as async?

Reply:
I don't believe so. I copied from `getCase` and `getDocument` above. Those seem to work correctly without `async`.